src/components/WorkOrder/New.js

```diff
@@ -150,24 +150,25 @@ class NewWorkOrder extends Component {
   submitForm = e => {
     e.preventDefault();
     this.setState({ errors: [], isSubmitting: true });
-    api
-      .workOrder()
-      .new(this.state.updatedFormData)
-      .then(res => {
-        this.setState({
-          isSubmitting: false,
-          isSubmitted: true,
-          newWorkOrder: res.data.record,
+    !this.state.isSubmitting &&
```

Comment:
good catch!

Reply:
It was a tricky one, but I'm glad that Toby reported it!